src/views/clusteroverview/overview/components/details-card/hooks/useDebounceCallback.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { debounce } from 'lodash';
+
+import { useDeepCompareMemoize } from '@openshift-console/dynamic-plugin-sdk/lib/utils/k8s/hooks/useDeepCompareMemoize';
```

Comment:
not sure if we can use this import (I think it was meant for internal use only), @vojtechszocs might know better than me on this

Reply:
not sure actually, if you find out, let everyone know :)

Reply:
I spoke with @vojtechszocs and he wasn't aware of any reason not to use it.